.gitignore

```diff
@@ -94,6 +94,7 @@ gradle-app.setting
 .gogradle
 
 operator/vendor
+vendor
```

Comment:
now that we have a single go.mod in the root...i've been getting `vendor` dirs created in the root on occasion

Reply:
merge master in and you'll get #88